src/middlewared/middlewared/plugins/etc.py

```diff
@@ -91,6 +72,9 @@ async def render(self, path, ctx):
 
 class EtcService(Service):
 
+    SYS_GROUPS = {}
```

Comment:
`SYS_GROUPS/USERS` variables aren't need here since they're being populated in the `internal_cache.py` file right?

Reply:
Ah shoot, yup :p